bundles/mapping/mapmodule/MapModuleClass.ol.js

```diff
@@ -1138,6 +1139,19 @@ export class MapModule extends AbstractMapModule {
         }
     }
 
+    /**
+     * Adds internal layer to map. Internal layers aren't listed in selected layers and are always above those.
+     * @method bringToTop
+     * @param {ol/layer/Layer} layer ol specific!
+     */
+    addInternalLayer (layerImpl) {
```

Comment:
Would something like "addOverlayLayer" be more descriptive name? It's not perfect either since there's already a concept of an "overlay" in OpenLayers but I think it could be better.